clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -149,6 +153,7 @@ private ClientConfig(
     this.maxAllowedKeyCntInBatchGetReq = maxAllowedKeyCntInBatchGetReq;
 
     this.daVinciClientForMetaStore = daVinciClientForMetaStore;
+    this.thinClientForMetaStore = thinClientForMetaStore;
```

Comment:
Do we need this option?
We know the other option: DaVinciBasedMeta won't work, so what is the point to introduce this config?